libraries/bezier-rs/Cargo.toml

```diff
@@ -13,9 +13,13 @@ homepage = "https://github.com/GraphiteEditor/Graphite/tree/master/libraries/bez
 repository = "https://github.com/GraphiteEditor/Graphite/tree/master/libraries/bezier-rs"
 documentation = "https://graphite.rs/libraries/bezier-rs/"
 
+[features]
+std = ["glam/std"]
```

Comment:
I needed this to run just the bezier-rs tests (with `cargo test -p bezier-rs --features=std`), because without it there were lots of undefined references to math functions.